web/profiles/custom/os2loop/themes/os2loop_theme/templates/content/node--os2loop-documents-collection--pdf.html.twig

```diff
@@ -115,28 +112,27 @@
 
   <div{{ content_attributes }}>
     <div class="row">
-      <div class="col-md-9">
+      <div class="pdf-wrapper w-60 float-left d-block px-3">
         {{ content.os2loop_documents_dc_content }}
-
-        {% if os2loop_documents_collection_tree|default(false) %}
-          <fieldset class="os2loop-documents--collection-toc">
-            <legend>{{ 'Documents in collection'|trans }}</legend>
-            {{ _self.table_of_contents(os2loop_documents_collection_tree, node) }}
-          </fieldset>
-          <div class="os2loop-documents--collection-documents">
-            {{ _self.document_content(os2loop_documents_collection_tree) }}
-          </div>
-        {% endif %}
       </div>
-      <div class="col-md-3">
+      <div class="pdf-wrapper w-25 float-right d-block pl-3">
         <div class="bg-secondary-loop">
           <div> {{ content.os2loop_shared_profession }}</div>
           <div> {{ content.os2loop_shared_tags }}</div>
           <div> {{ content.os2loop_shared_subject }}</div>
           <div> {{ content.os2loop_documents_document_autho }}</div>
-          <div> {{ content.flag_os2loop_favourite }}</div>
         </div>
       </div>
     </div>
   </div>
+
+  {% if os2loop_documents_collection_tree|default(false) %}
+    <fieldset class="os2loop-documents--collection-toc">
+      <h2>{{ 'Documents in collection'|trans }}</h2>
+      {{ _self.table_of_contents(os2loop_documents_collection_tree, node) }}
+    </fieldset>
+    <div class="os2loop-documents--collection-documents">
+      {{ _self.document_content(os2loop_documents_collection_tree) }}
+    </div>
+  {% endif %}
```

Comment:
Is it a design choice or a customer request to move the collection table of contents?

Reply:
Thats a design choice